tox.ini

```diff
@@ -4,7 +4,9 @@ skipsdist = True
 envlist = py310,py39,py38,py37,pep8,black,twine-check,mypy,isort
 
 [testenv]
-passenv = GITLAB_IMAGE GITLAB_TAG
+# NOTE: PY_COLORS, NO_COLOR, and FORCE_COLOR are used by pytest, so pass them through if
+# set. https://docs.pytest.org/en/6.2.x/reference.html
```

Comment:
I was trying to explain where PY_COLORS is coming and why we are passing it. So future people will know why it is there and can hopefully make an informed decision on changing things. I know sometimes in code it can be worrisome to change things as we don't know why something is there and what it will break if it is changed.

But I can delete the comment too.

Reply:
I guess I meant that we might need to keep updating this then as we add more variables to pass, and that even adding it here it's possible it started changing for other tools not just pytest :) But I didn't check. Feel free to resolve this otherwise :)

Reply:
I went ahead and removed it.